.github/workflows/playwright.yml

```
@@ -27,7 +27,14 @@ jobs:
     - name: Run Playwright tests
       env:
         IDENTITY: ${{ secrets.DAGIT_TEST_IDENTITY }}
+        IDENTITY2: ${{ secrets.DAGIT_TEST_IDENTITY2 }}
       run: make test
+          - uses: actions/upload-artifact@v4
```

actionlint

could not parse as YAML: yaml: line 32: mapping values are not allowed in this context

(syntax-check)